clappr/clappr-ios
# Update project to swift 5.0.1 and Xcode 10.2

# Goal

Migrate the project to Swift 5.0.1 and Xcode 10.2.

# How to test
1. Run the tests
2. Run the example app and check the expected features still work
